src/qtism/data/storage/xml/marshalling/MarshallerFactory.php

```diff
@@ -375,18 +376,23 @@ public function isWebComponentFriendly()
      *
      * @param DOMElement|QtiComponent $object A QtiComponent or DOMElement object you want to get the corresponding Marshaller object.
      * @param array $args An optional array of arguments to be passed to the Marshaller constructor.
+     *
      * @return Marshaller The corresponding Marshaller object.
-     * @throws RuntimeException If no Marshaller object can be created for the given $object.
      * @throws MarshallerNotFoundException If no Marshaller mapping is set for a given $object.
-     * @throws InvalidArgumentException If $object is not a QtiComponent nor a DOMElement object.
+     * @throws InvalidArgumentException    If $object is not a QtiComponent nor a DOMElement object.
+     *
+     * @throws RuntimeException            If no Marshaller object can be created for the given $object.
      */
-    public function createMarshaller($object, array $args = [])
+    public function createMarshaller($object, array $args = []): Marshaller
     {
         if ($object instanceof QtiComponent) {
             // Asking for a Marshaller...
             $qtiClassName = $object->getQtiClassName();
 
-            if ($this->isWebComponentFriendly() === true && in_array($qtiClassName, Marshaller::$webComponentFriendlyClasses)) {
+            if ($this->isWebComponentFriendly() === true && in_array(
+                    $qtiClassName,
+                    Marshaller::$webComponentFriendlyClasses
+                )) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place